src/components/ActivityForm.js

```diff
@@ -94,6 +86,11 @@ export default class ActivityForm extends React.Component {
           <button>Add Activity</button>
         </form>
       </div>
-    )
+    );
   }
+}
+
+ActivityForm.propTypes = {
+  activity: PropTypes.object,
```

Comment:
activity seems to be reusable prop in some of the places. You could create separate folder in `src` called `prop-types` and there defined shape of the activity and reuse in all of the occurances.

Reply:
ah. This is the class based component. 
So, in similar way like in example `constructor` or `render` you can create `static` things. Inside the class, above constructor, make:
```
static propTypes = {
....
}
```

Reply:
instead of this approach